vip-plugins/vip-plugins.php

```diff
@@ -200,7 +202,9 @@ function wpcom_vip_site_option_active_sitewide_plugins( $value, $option ) {
 		$value = array_merge( $value, $code_plugins );
 	}
 
-	ksort( $value );
+	if ( ! empty( $value ) ) {
```

Comment:
Do we need to do the same thing in `wpcom_vip_pre_update_option_active_plugins` and `wpcom_vip_pre_update_site_option_active_sitewide_plugins`?

Reply:
Good thinking. Better to be safe I think.